CMakeLists.txt

```
@@ -70,14 +70,21 @@ add_custom_target(copyShaders ALL SOURCES ${SHADER_FILES})
 # Add dependencies
 add_subdirectory(libs)
 
+if(SDFLIB_USE_ENOKI)
+    target_link_libraries(${PROJECT_NAME} PUBLIC enoki)
+    target_link_libraries(${PROJECT_NAME} PUBLIC fcpw)
+    target_compile_definitions(${PROJECT_NAME} PUBLIC -DENOKI_AVAILABLE)
+endif()
+
+if(SDFLIB_USE_ASSIMP)
+    target_link_libraries(${PROJECT_NAME} PUBLIC assimp)
+    target_compile_definitions(${PROJECT_NAME} PUBLIC -DASSIMP_AVAILABLE)
```

Minor comment: given that this is used in a public header, it may be safer to avoid macro conflicts to prefix with a sdflib-specific prefix, like:
```suggestion
    target_compile_definitions(${PROJECT_NAME} PUBLIC -DSDFLIB_ASSIMP_AVAILABLE)
```